cli/hid.ts

```diff
@@ -2,14 +2,20 @@ import HF2 = pxt.HF2
 import U = pxt.U
 import * as nodeutil from './nodeutil';
 
+function useWebUSB() {
+    return !!pxt.appTarget.compile.webUSB
```

Comment:
can compile be missing?

Reply:
no - https://github.com/Microsoft/pxt/blob/master/pxtlib/main.ts#L91